### PR TITLE
[nnbd] Various fixes

### PR DESCRIPTION
**Pull request by blasten**
* Cleans up some of the code, so it's consistent.  
* Fixes some checks. (hopefully all)

### CODE REVIEW

File: packages/connectivity/connectivity/test/connectivity_test.dart [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.8
+// @dart = 2.9
```
Comment (cyanglaz) [inline packages/connectivity/connectivity/test/connectivity_test.dart:5]: connectivity hasn't moved to nnbd yet, my migration PR will handle this. Do you want to maybe revert it for now?

File: packages/url_launcher/url_launcher_windows/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## 0.1.0-nullsafety
+
+* Migrate to null-safety.
+
+## 0.0.1+1
+
+* Update README to reflect endorsement.
```
Comment (cyanglaz) [inline packages/url_launcher/url_launcher_windows/CHANGELOG.md:7]: Duplicated 0.0.1+1. Looks like a merge error.

**Review by cyanglaz — CHANGES_REQUESTED** [review]

File: packages/connectivity/connectivity/test/connectivity_test.dart (hunk 1 above)
Comment (blasten) [reply packages/connectivity/connectivity/test/connectivity_test.dart:5]: `@dart = 2.9` is non-null safety.  `@dart = 2.10` enables null safety.

File: packages/url_launcher/url_launcher_windows/CHANGELOG.md (hunk 2 above)
Comment (blasten) [reply packages/url_launcher/url_launcher_windows/CHANGELOG.md:7]: Good catch

**Comment by blasten** [comment]
Some dependency resolution will fail for this patch, but it should be fixed after this patch is merged. This is due to some git dependencies depending on path dependencies. After this change, everything should be fixed.

**Review by cyanglaz — APPROVED** [review]
LGTM

**Pull request merged**